destination/destination.go

```diff
@@ -81,8 +142,15 @@ func (d *Destination) Open(ctx context.Context) error {
 }
 
 // Write writes a record into a Destination.
-func (d *Destination) Write(ctx context.Context, record sdk.Record) error {
-	return d.writer.Write(ctx, record)
+func (d *Destination) Write(ctx context.Context, records []sdk.Record) (int, error) {
+	for i, record := range records {
```

Comment:
is it possible to do it using batch insert?

Reply:
No, it's not. NATS PubSub doesn't support batch inserts